Topics/Tech_Stacks/Flutter.md

```diff
@@ -0,0 +1,79 @@
+# Learning Flutter
+
+## Table of contents
+
+#### [What is Flutter](#what-is-flutter)
+
+#### [Why Flutter? Pros and Cons](#why-flutter-pros-and-cons)
+
+#### [Creating your first flutter app](#creating-your-first-flutter-app)
+
+## What is Flutter?
+
+Flutter is an open source framework developed and supported by Google. Frontend and full-stack developers use Flutter to build an application’s user interface (UI) for multiple platforms with a single codebase.
```

Comment:
Could you give some examples of products or companies that use flutter for their development? It would be great to showcase what flutter is capable of

Reply:
+1 on this, great idea!